StyleGuidelines.md

```diff
@@ -42,6 +42,7 @@ In order to provide clean and consistent code, please follow the style guideline
 - [Best Practices](#best-practices)
   - [Named Parameters Instead of Positional Parameters](#named-parameters-instead-of-positional-parameters)
   - [No Cmdlet Aliases](#no-cmdlet-aliases)
+  - [Capitalized cmdlets, Function Names and Pester Assertions](#capitalized-cmdlets-function-names-and-pester-assertions)
```

Comment:
The entry in the TOC needs to updated to reflect the updated title of the section